### DISCUSSION TITLE
Embedded Document Management Support

### DISCUSSION THREAD

**Discussion by harshalizode** [dymmond/mongoz#51] [Ideas]
Hello,
We are going to implement embedded document management support in the Mongoz, and we need assistance from you to finalize required API Specifications compliant to existing API Specifications of Mongoz.

**Comment by tarsil** [comment]
@harshalizode are you adding/contributing this to Mongoz or simply for you internally? Well, what help do you need?

1. The [EmbeddedDocument](https://mongoz.tarsild.io/embedded-documents/?h=embedded) is a simple Pydantic BaseModel but with Mongoz fields. It does not query any document, it does not have a meta class, it does not have any db operations, its embedded as the name suggests.

What do you require?

**Reply by harshalizode** [reply]
We require a program specification for how to integrate the embedded document management with Mongoz . With the MongoDB we are able to manage the embedded documents. So, we'll add the MongoDB capability to Mongoz. We help to figuring out how to do it with the Mongoz compliance.

**Comment by harshalizode** [comment]
@tarsil is any update on this?

**Comment by harshalizode** [comment]
I have created the workaround for the embedded document to update the in the base document.
I have used the **BulkWrite**  method from the pymongo that can we inherit in the our Query Manager to per form the bulk operations.
